forch/http_server.py

```diff
@@ -78,7 +78,10 @@ def __init__(self, port, config=None):
         self._config = config
         self._paths = {}
         self._server = None
-        self._root_path = config.http_root if config else 'public'
+        if config and config.http_root:
```

Comment:
self._root_path = config.http_root if config and config.http_root else 'public'

Reply:
Done.